src/wwwroot/js/genpage/helpers/generatehandler.js

```diff
@@ -199,6 +199,10 @@ class GenerateHandler {
                         }
                         this.setImageFor(imgHolder, data.gen_progress.preview);
                     }
+                    if (data.gen_progress.preview && imageFullView.isOpen() && imageFullView.imgElement && imageFullView.currentBatchId == thisBatchId)
+                    {
```

Comment:
shouldn't be a newline here for JS code

Reply:
I had the newline there so I could set breakpoints and verify that the arguments were good.  Do I need to change it back?